### PR TITLE
Document comment support in Delta Lake

### PR DESCRIPTION
**Pull request by colebow**
## Description

Docs for #12971

> Is this change a fix, improvement, new feature, refactoring, or other?

Docs only

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

**Review by mosabua — APPROVED** [review]
Nice and easy ;-)

**Review by findinpath — APPROVED** [review]

File: docs/src/main/sphinx/connector/delta-lake.rst [hunk 1]
```diff
@@ -343,6 +343,7 @@ statements, the connector supports the following features:
 * :doc:`/sql/drop-schema`
 * :doc:`/sql/show-create-schema`
 * :doc:`/sql/show-create-table`
+* :doc:`/sql/comment`
```
Comment (findinpath) [inline docs/src/main/sphinx/connector/delta-lake.rst:346]: Note that Iceberg & Hive supports  also `COMMENT`.
(Mongo is already documented)

Comment (mosabua) [reply docs/src/main/sphinx/connector/delta-lake.rst:346]: @colebow can check and send PRs

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**